hw7/task34.py

```diff
@@ -0,0 +1,43 @@
+"""
+Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм.
+Поскольку разобраться в его кричалках не настолько просто,
+насколько легко он их придумывает, Вам стоит написать программу.
+Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв)
+в каждой фразе стихотворения одинаковое.
+Фраза может состоять из одного слова, если во фразе несколько слов,
+то они разделяются дефисами. Фразы отделяются друг от друга пробелами.
+Стихотворение  Винни-Пух вбивает в программу с клавиатуры.
+В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и
+“Пам парам”, если с ритмом все не в порядке
+
+*Пример:*
+
+**Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да
+    **Вывод:** Парам пам-пам
+"""
+
+
+def vowels_count(phrase):
+    vowel_set = ["А", "Е", "Ё", "И", "О", "У", "Ы", "Э", "Ю", "Я"]
+    rslt = 0
+    for symbol in phrase:
+        if symbol.upper() in vowel_set:
```

Comment:
Логичней применять метод `upper` один раз, чем каждую итерацию.
```suggestion
    for symbol in phrase.upper():
        if symbol in vowel_set:
```